common/app/views/support/ContentOldAgeDescriber.scala

```diff
@@ -34,7 +34,7 @@ class ContentOldAgeDescriber {
       if (pubDate.isBefore(DateTime.now().minusDays(warnLimitDays))) {
         val ageMillis = DateTime.now().getMillis - pubDate.getMillis
         val years = TimeUnit.MILLISECONDS.toDays(ageMillis) / 365
-        val months = TimeUnit.MILLISECONDS.toDays(ageMillis) / 28
+        val months = TimeUnit.MILLISECONDS.toDays(ageMillis) / 30
```

Comment:
Why not 31? 😝

Reply:
Technically should be `365.24/12`, ie: `30.4367` :)